apps/em/em-ped-api/aat.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
```

Comment:
this does nothing, unless u patch it somewhere.

Reply:
My understanding was this was the missing bit. In which case not sure why the PAI is not been deployed on AAT. Any clue?